src/go/k8s/tests/e2e-v2/upgrade-values-check/01-upgrade-redpanda.yaml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
I don't understand this test case step. If the content of `src/go/k8s/tests/e2e-v2/upgrade-values-check/01-upgrade-redpanda.yaml` is identical to `src/go/k8s/tests/e2e-v2/upgrade-values-check/00-create-redpanda.yaml` the kuttle test will not update Redpanda resource, right?

Reply:
I am still working on the testing.